.env.sample

```diff
@@ -10,3 +10,5 @@ FB_APP=1583385451706458
 
 GOOGLE_API_V3_KEY=AIzaSyCDbxxvbl0Zj6af28IfWjJ-S0SKxwDXnkc
 GOOGLE_TAG_MANAGER_ID=GTM-MZ2S75K
+GOOGLE_ANALYTICS=UA-46158613-2
```

Comment:
Are these already surfaced in the browser / public?

Reply:
these are only for local development. For prod and demo they get set in circle